tools/add_triage_signature.py

```diff
@@ -96,9 +102,16 @@ def get_remote_archive(tar_url):
     return nestedarchive.RemoteNestedArchive(tar_url, init_download=True)
 
 
+class FailedToGetLogsTarException(Exception):
+    pass
+
+
 def get_triage_logs_tar(triage_url, cluster_id):
-    tar_url = f"{triage_url}/cluster_{cluster_id}_logs.tar"
-    return get_remote_archive(tar_url)
+    try:
+        tar_url = f"{triage_url}/cluster_{cluster_id}_logs.tar"
+        return get_remote_archive(tar_url)
+    except requests.exceptions.HTTPError as e:
```

Comment:
same as above. 5xx should be noticeable

Reply:
Don't want behavior changes, I only moved it from the signatures to here to simplify the code. Can change this in a different PR